WordPress/src/main/java/org/wordpress/android/ui/utils/PaddingBackgroundColorSpan.kt

```diff
@@ -0,0 +1,40 @@
+package org.wordpress.android.ui.utils
+
+import android.graphics.Canvas
+import android.graphics.Paint
+import android.graphics.Rect
+import android.text.style.LineBackgroundSpan
+import kotlin.math.roundToInt
+
+/**
+ * Creates BackgroundColorSpan with padding
+ * Credits: https://medium.com/@tokudu/android-adding-padding-to-backgroundcolorspan-179ab4fae187
```

Comment:
@ashiagr - I never noticed the "non" spacing before, but this is a good hack. Nice find and enhancement